seatunnel-connectors-v2/connector-file/connector-file-base/pom.xml

```diff
@@ -73,5 +75,12 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.flink</groupId>
```

Comment:
If we use flink hadoop jar, the job will failed in spark. So we should set the `scope` of `hadoop` to `provided`.

Reply:
Yes, you are right, I will change it.

Reply:
Resolved, please review.